scico/loss.py

```diff
@@ -80,7 +87,37 @@ def __call__(self, x: Union[JaxArray, BlockArray]) -> float:
         Args:
             x: Point at which to evaluate loss.
         """
-        raise NotImplementedError
+        if self.lfunc is None:
+            raise NotImplementedError(
+                "Functional l is not defined and __call__ has" " not been overridden"
+            )
+        return self.scale * self.lfunc(self.A(x) - self.y)
+
+    def prox(
+        self, v: Union[JaxArray, BlockArray], lam: float, **kwargs
+    ) -> Union[JaxArray, BlockArray]:
+        r"""Scaled proximal operator of loss function.
+
+        Evaluate scaled proximal operator of this loss function, with
+        scaling :math:`\lambda` = `lam` and evaluated at point
+        :math:`\mb{v}` = `v`. If :meth:`prox` is not defined in a derived
+        class, and if operator :math:`A` is the identity operator, the
```

Comment:
inserting a `then` between `operator` and `the` makes it easier to read in my opinion.

Reply:
Agreed. Done.